Snapshots.md

```diff
@@ -7,7 +7,7 @@ layout: default
 
 - [Current development version](https://dafny.org/dafny)
 - [Latest release snapshot](https://dafny.org/latest)
-- [v4.6.0](https://dafny.org/v4.6.0)
+- [v4.8.1](https://dafny.org/v4.8.1)
```

Comment:
Looks like we're overwriting instead of appending in a script somewhere

Reply:
Script looks good: 
`sed -i -e "szlatest)zlatest)\n- [$V](https://dafny.org/$V)z" Snapshots.md`

It's as if it did not have the latest changes checked out, but looking at the script I don't see an issue.